extensions/localPublish/src/WebSocketServer.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import http from 'http';
+
+import portfinder from 'portfinder';
+import express, { Request, Response } from 'express';
+import { Server as WSServer } from 'ws';
+
+interface WebSocket {
+  close(): void;
+  send(data: string, cb?: (err?: Error) => void): void;
+}
+
+export class WebSocketServer {
```

Comment:
I'd give this a better name to reflect on what the purpose of the web socket server is.

Reply:
Yep. Changing it to runtimeLogServer